tk_toolchain/cmd_line_tools/tk_build_qt_resources/__main__.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2024 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+from tk_toolchain.cmd_line_tools.tk_build_qt_resources import main
```

Comment:
Can we use local import instead?

Reply:
Any reasons why? I see other cmd scripts using this same pattern. However, it is assumed the scripts are executed from the repo's root. I'm not against it too.